### PR TITLE
[*]BO: New navigation & header

### PR DESCRIPTION
**Pull request by Shudrum**
The new tabs & header are now here!

Some things are now broken and will be up ASAP:
- [ ] The element where modules icons are placed on the header will be put at the right place
- [ ] All the notifications will be put on the new single notifications center
- [ ] The loader will be different
- [ ] The multishop select will be put again

### CODE REVIEW

File: install-dev/data/db_structure.sql [hunk 1]
```diff
@@ -1854,6 +1854,7 @@ CREATE TABLE `PREFIX_tab` (
   `position` int(10) unsigned NOT NULL,
   `active` tinyint(1) NOT NULL DEFAULT 1,
   `hide_host_mode` tinyint(1) NOT NULL DEFAULT '0',
+  `icon` varchar(32) DEFAULT '',
```
Comment (xGouley) [inline install-dev/data/db_structure.sql]: report this in upgrade sql fil (1.7.0.0.sql) ?

File: classes/controller/AdminController.php [hunk 2]
```diff
@@ -2589,6 +2586,7 @@ public function setMedia()
         //Bootstrap
         $this->addCSS(__PS_BASE_URI__.$this->admin_webpath.'/themes/'.$this->bo_theme.'/css/'.$this->bo_css, 'all', 0);
         $this->addCSS(__PS_BASE_URI__.$this->admin_webpath.'/themes/'.$this->bo_theme.'/css/vendor/titatoggle-min.css', 'all', 0);
+        $this->addCSS('https://fonts.googleapis.com/icon?family=Material+Icons', 'all', 0);
```
Comment (xGouley) [inline classes/controller/AdminController.php]: Question:
http / https ? both works, why https ?

File: install-dev/data/db_structure.sql (hunk 1 above)
Comment (Shudrum) [inline install-dev/data/db_structure.sql]: When the final navigation will be finished (maybe some things will move), the upgrade will be done for this, and all the modifications done on the xml files.

File: classes/controller/AdminController.php (hunk 2 above)
Comment (Shudrum) [inline classes/controller/AdminController.php]: Because this line will be removed when the UI Kit will come. This is temporary.

**Comment by xGouley** [comment]
:+1: for me except sass part not reviewed.

**Comment by Shudrum** [comment]
Sass is as clean as before 😷
All will be re-indented soon.

Thank you !

**Comment by Shudrum** [comment]
Thank you!

**Comment by xGouley** [comment]
You are welcome! :bouquet:

**Pull request merged**